library/objective-c/BUILD

```diff
@@ -16,6 +16,7 @@ objc_library(
     hdrs = [
         "EnvoyEngine.h",
     ],
+    module_name = "EnvoyEngine",
```

Comment:
Previously this name didn't matter at all because we never imported it directly, it happened implicitly with a bridging header